combat-trainer.lic

```diff
@@ -1695,7 +1730,7 @@ class SpellProcess
 
     game_state.casting_regalia = data['abbrev'].casecmp('REGAL').zero?
 
-    prepare?(data['abbrev'], data['mana'], data['symbiosis'], command, data['tattoo_tm'])
+    prepare?(data['abbrev'], data['mana'], data['symbiosis'], command, data['tattoo_tm'], data['runestone_name'], data['runestone_tm'])
```

Comment:
❌ Won't this be a breaking change?

We'll need to split `combat-trainer` into a separate PR and merge that one after a few days.